AutoRest/Generators/NodeJS/NodeJS/ClientModelExtensions.cs

```diff
@@ -242,6 +242,28 @@ private static string ValidatePrimaryType(this PrimaryType primary, IScopeProvid
             }
         }
 
+        /// <summary>
+        /// Returns the TypeScript type string for the specified primary type
+        /// </summary>
+        /// <param name="primary">primary type to query</param>
+        /// <returns>The TypeScript type correspoinding to this model primary type</returns>
+        private static string PrimaryTSType(this PrimaryType primary) {
+            if (primary == PrimaryType.Boolean)
+                return "boolean";
+            else if (primary == PrimaryType.Double || primary == PrimaryType.Int || primary == PrimaryType.Long)
+                return "number";
+            else if (primary == PrimaryType.String)
+                return "string";
+            else if (primary == PrimaryType.Date || primary == PrimaryType.DateTime)
+                return "Date";
+            else if (primary == PrimaryType.Object)
+                return "Object";   // TODO: test this
```

Comment:
`Object` usually isn't what users actually want. If users are unsure of what a value can be, they usually go with `any` which I'd suggest in this case.